docs/cloud-providers/aws-sdk.md

```diff
@@ -25,6 +25,12 @@ with the naming guidelines for RPC client spans.
 
 <!-- prettier-ignore-start -->
```

Comment:
Depends on what we plan to do with `prettier`.  We use BOTH prettier *and* markdown lint.  We have not enabled prettier for all markdown yet, but when we do it'll break the snippet generation checking as it would pretty-print markdown after generataion.


Option 1 - I could enable prettier on *all* markdown and force it to run after either attribute registry generation *or* table-generation.

Option 2 - I can turn it off here and we can decide to stop using prettier.

Option 3 - We leave this as is and decide on prettier later.

Reply:
let's do option 3